### ISSUE TITLE
[feature request] After editing a task and bigboard reloads page, scroll the page back to where you were.

### ISSUE THREAD

**Issue by xekon** [BlueTeck/kanboard_plugin_bigboard#24]
Currently if your showing a good number of projects in big board, and you scroll waaaaay down and click "edit the task" on a card, you will see in the background that the bigboard stays where you scrolled it to until the moment you hit save on your edit, once you do that the page is back at the top, so you have to scroll back down to where you were.

Would be great if the it would scroll back to where you were on the page.

**Comment by BlueTeck** [comment]
Unfortunately I dont use Kanboard anymore, therefore I wont improve/change my plugins. But I am open for pull requests.

**Comment by xekon** [comment]
fair enough! thanks, may I ask what kanban style board your currently using? (I have been reviewing alternatives)

**Comment by BlueTeck** [comment]
We are using Azure Boards now, but Kanboard was much better... just a bad company policy (going full Microsoft 365)...

**Comment by xekon** [comment]
OUCH! I feel for ya. thanks again! we are still using the bigboard

**Comment by xekon** [comment]
going to leave this open incase somebody else sees it. that decides to fork or clone bigboard and extend it.
